src/common/api/worker/offline/migrations/sys-v104.ts

```diff
@@ -0,0 +1,14 @@
+import { OfflineMigration } from "../OfflineStorageMigrator.js"
+import { OfflineStorage } from "../OfflineStorage.js"
+import { SqlCipherFacade } from "../../../../native/common/generatedipc/SqlCipherFacade.js"
+import { addValue, migrateAllListElements } from "../StandardMigrations.js"
+import { PushIdentifierTypeRef } from "../../../entities/sys/TypeRefs.js"
+import { PushIdentifierAppType } from "../../../../native/main/NativePushServiceApp.js"
+
+export const sys104: OfflineMigration = {
+	app: "sys",
+	version: 104,
+	async migrate(storage: OfflineStorage, _: SqlCipherFacade) {
+		await migrateAllListElements(PushIdentifierTypeRef, storage, [addValue("app", PushIdentifierAppType.Integrated)])
```

Comment:
this might be wrong. The user might upgrade when there are already separate push identifiers.
I would recommend you to just delete them all from cache.